pkg/controllers/cspi-controller/handler.go

```diff
@@ -241,6 +227,51 @@ func (c *CStorPoolInstanceController) updateStatus(cspi *cstor.CStorPoolInstance
 	return nil
 }
 
+// updateROMode sets/unsets the pool readonly mode property. It does the following changes
+// 1. If pool used space reached to roThresholdLimit then pool will be set to readonly mode
+// 2. If pool was in readonly mode if roThresholdLimit/pool expansion was happened then it
+//    unsets the ReadOnly Mode.
+// NOTE: This function must be invoked after having the updated cspiStatus information
```

Comment:
:100: for comments...  
Trying to understand the `NOTE` section of comment...

Reply:
NOTE: Is trying to convey that this function should be called only after getting the updated info from the pool(by using zpool/zfs command). If we call this function with stale/outdated capacity values then the behaviour will not be as expected.